build/__nodeVersions.sh

```diff
@@ -1,16 +1,16 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-NODE_RUNTIME_BASE_TAG='20230407.4'
+NODE_RUNTIME_BASE_TAG='20230416.1'
 YARN_VERSION='1.22.15'
 YARN_MINOR_VERSION='1.17'
 YARN_MAJOR_VERSION='1'
 NODE6_VERSION='6.17.1'
 NODE8_VERSION='8.17.0'
 NODE10_VERSION='10.23.0'
-NODE12_VERSION='12.22.11'
-NODE14_VERSION='14.21.2'
-NODE16_VERSION='16.19.0'
-NODE18_VERSION='18.14.0'
+NODE12_VERSION='12.22.12'
```

Comment:
We no longer create node 12 runtime versions... hopefully that is fine with them...

Reply:
we will keep the stretch changes as we are running out of time to meet the deadline.